pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -334,97 +256,7 @@ def _training_step(
             if result_collection is None:
                 return
 
-        closure_loss = None
-        loss = None
-        if self.trainer.lightning_module.automatic_optimization:
-            # accumulate loss. if accumulate_grad_batches==1, no effect
-            closure_loss = result_collection.minimize / self.trainer.accumulate_grad_batches
-            # the loss will get scaled for amp. avoid any modifications to it
-            loss = closure_loss.detach().clone()
-        return AttributeDict(closure_loss=closure_loss, loss=loss, result_collection=result_collection)
-
-    def _optimizer_step(
```

Comment:
reviewer: these removed methods below are all specific to automatic optimization and get moved to the OptimizerLoop